### ISSUE TITLE
DOC: Add documentation section for flattener/nester elements

### ISSUE THREAD

**Issue by drewhk** [akka/akka#16801]

**Comment by drewhk** [comment]
Don't forget to explain the dangers of deadlocks, and to prefer static graphs when possible.

**Comment by rkuhn** [comment]
splitWhen / groupBy / …

**Comment by rkuhn** [comment]
While this was written with something else in mind we do also need to describe the SubFlow better.

**Comment by johanandren** [comment]
Not sure what this is about, but groupBy seems to have gotten a warning in the docs. Closing for now, please reopen if you disagree.

**Comment by drewhk** [comment]
I disagree ;) The point was to add a chapter in the docs about flattener-nester ops. We have things like graphs, rate-based ops, integration ops, custom ops, but stream-of-stream kind of stuff is not really documented outside API docs.

**Comment by johanandren** [comment]
Ok, that explains, thanks.
